src/components/nav/updateStateToUseOnBackAndForwardNav.tsx

```diff
@@ -0,0 +1,14 @@
+import { history } from "./history";
+import { navTo } from "./navTo";
+import { renderApp } from "../state/renderApp";
+import { UnregisterCallback } from "history";
+import { doNothing } from "../../utils/doNothing";
+
+//TODO: Undo the global state here
+export let previousHistoryListener:UnregisterCallback = doNothing;
+export const updateStateToUseOnBackAndForwardNav = state => {
```

Comment:
this is to make sure that we force a re-render of the app when the user clicks back / forward; and the case of looking for "POP" is to prevent it from re-rendering twice on a normal navTo that is _not_ a back/forward